tests/benchdnn/dnn_types.hpp

```diff
@@ -81,6 +82,7 @@ struct attr_t {
     static int policy2mask(int arg, policy_t policy, int ndims = -1,
             dnnl_primitive_kind_t prim_kind = dnnl_undefined_primitive,
             bool has_groups = false);
+    static dnnl_quantization_kind_t policy2quantization_kind(policy_t policy);
```

Comment:
It doesn't look like it should be `static` so far.

Reply:
I followed existing style of not requiring an object for those conversions.
I am fine either way to be honest, but why make this one different than policy2mask?

Reply:
policy2mask was converted to `static` to cover usages outside of attributes. A new function doesn't seem to do be used outside of attrs, and then it could reuse some new methods introduced to avoid code duplication.